tests/robot-tests/tests/admin_and_public/bau/archive_publication.robot

```diff
@@ -341,15 +349,16 @@ Check public Find stats page and check archive-publication is no longer archived
     [Documentation]    Failing due to https://dfedigital.atlassian.net/browse/EES-4269
     [Tags]    Failing
 
+    # TODO EES-6063 - Remove this
     user checks publication is on find statistics page    ${PUBLICATION_NAME_SUPERSEDE}
     user checks publication is on find statistics page    ${PUBLICATION_NAME_ARCHIVE}
 
 Check public archive-publication release page displays correctly after being unarchived
     [Documentation]    Failing due to https://dfedigital.atlassian.net/browse/EES-4269
     [Tags]    Failing
 
-    user clicks link    ${PUBLICATION_NAME_ARCHIVE}
-
+    user navigates to public release page    ${PUBLICATION_URL_ARCHIVE}    ${PUBLICATION_NAME_ARCHIVE}
+    ...    ${RELEASE_NAME_ARCHIVE}
```

Comment:
I think the line immediately after this can be removed now because it's already included by `user navigates to public release page`.

```
user waits until h1 is visible    ${PUBLICATION_NAME_ARCHIVE}    %{WAIT_MEDIUM}
```